python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -417,8 +419,23 @@ def fetch_cancelable_run_ids_for_asset_backfill(instance: DagsterInstance, backf
     return [run.run_id for run in backfill_runs if run.status in CANCELABLE_RUN_STATUSES]
 
 
+def _check_workspace_is_loadable(context: IWorkspace, logger: logging.Logger) -> None:
+    location_entries_by_name = {
+        location_entry.origin.location_name: location_entry
+        for location_entry in context.get_workspace_snapshot().values()
+    }
+
+    for location_name, location_entry in location_entries_by_name.items():
+        if location_entry.load_error:
+            logger.warning(
```

Comment:
Here we just throw a warning instead of erroring. This is because if just one location in the workspace is unloadable, we don't want to terminate unaffected backfills